compiler/commands.nim

```diff
@@ -735,7 +735,11 @@ proc processSwitch*(switch, arg: string, pass: TCmdLinePass, info: TLineInfo;
   of "nep1":
     processOnOffSwitchG(conf, {optCheckNep1}, arg, pass, info)
   of "cppcompiletonamespace":
-    expectNoArg(conf, switch, arg, pass, info)
+    if conf != nil:
```

Comment:
``conf`` cannot be nil here.

Reply:
good points, I will resolve both later!

btw I noticed that conf is checked against `nil` often above, like in `header` option or so.
I was wondering, can it be nil anyway?

Reply:
Nah. Not anymore.